controller/src/endpoints/post.py

```diff
@@ -1,37 +1,239 @@
 import logging
+import os
 
 from flask import request
 from flask_restx import Resource, fields, Namespace
-from flask_jwt_extended import jwt_required
+from flask_jwt_extended import jwt_required, get_jwt_identity
 
 from ..database.queries import Queries as db
+from bson.objectid import ObjectId
 from ..utils.request import send_request
 from ..utils.apps import Services
+from ..utils.apps import Url
+
+from datetime import datetime
 
 
 log = logging.getLogger('POST')
 
 api = Namespace('post')
 
+post_model = api.model('Post', {
+    '_id': fields.String(description="User ID of the post creator", example="671f880f5bf26ed4c9f540fd"),
+    '_user_id': fields.String(description="User ID of the post creator", example="671f880f5bf26ed4c9f540fd"),
+    'description': fields.String(description="Post description", example="My first post!"),
+    'images_urls': fields.List(fields.String, description="List of image URLs", example=["url1", "url2"]),
+    'timestamp': fields.String(description="ISO formatted timestamp", example="2023-12-01T12:34:56Z"),
+    'reactions_count': fields.Raw(description="Reaction counts (e.g., likes)", example={"likes": 0}),
+    'location': fields.String(description="Location of the post", example="New York, NY"),
+})
+
+post_list_model = api.model('PostList', {
+    'posts': fields.List(fields.Nested(post_model), description="List of posts")
+})
 
-@api.route('/')
+@api.route('/posts')
 class Post(Resource):
+    @api.doc(
+        description="Fetch posts based on optional filters (e.g., user ID, pagination).",
```

Comment:
Wydaje mi się że niepotrzebne jest to description w @api.doc, bo to samo napisane jest w docstringu w endpoincie, a docstring jest wyświetlany w swaggerze, więc mozna usunąć tę linijkę z description w @api.doc